WPFHospitalEditor/Controller/Interface/ISchedulingServerController.cs

```diff
@@ -0,0 +1,11 @@
+﻿using HealthcareBase.Model.Schedule.SchedulingPreferences;
+using HospitalWebApp.Dtos;
+using System.Collections.Generic;
+
+namespace WPFHospitalEditor.Controller.Interface
+{
+    public interface ISchedulingServerController
+    {
+        public List<RecommendationDto> GetAppointments(RecommendationRequestDto recommendationRequestDto);
+    }
```

Comment:
Do you need public access modifier here?

Reply:
No, deleted